src/go/rpk/pkg/cli/container/start.go

```diff
@@ -42,29 +42,72 @@ type node struct {
 	addr string
 }
 
-func collectFlags(args []string, flag string) []string {
-	flags := []string{}
-	i := 0
-	for i < len(args)-1 {
-		if args[i] == flag {
-			flags = append(flags, args[i], args[i+1])
-		}
-		i++
-	}
-	return flags
+type clusterPorts struct {
+	adminPorts  []uint
+	kafkaPorts  []uint
+	proxyPorts  []uint
+	rpcPorts    []uint
+	schemaPorts []uint
 }
 
 func newStartCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+	const (
+		flagSRPorts    = "schema-registry-ports"
+		flagProxyPorts = "proxy-ports"
+		flagKafkaPorts = "kafka-ports"
+		flagAdminPorts = "admin-ports"
+		flagRPCPorts   = "rpc-ports"
+		flagAnyPort    = "any-port"
+	)
 	var (
 		nodes     uint
 		retries   uint
 		image     string
 		pull      bool
 		noProfile bool
+
+		aPorts  []string
+		kPorts  []string
+		pPorts  []string
+		rPorts  []string
+		srPorts []string
+		anyPort bool
 	)
 	command := &cobra.Command{
 		Use:   "start",
 		Short: "Start a local container cluster",
+		Long: `Start a local container cluster.
+
+This command utilizes Docker to initiate a local container cluster. Use the '--nodes'/'-n' flag to specify the number of
+nodes.
+
+The initial broker starts on default ports, with subsequent brokers' ports offset by 1000. You can use the listeners 
+flag to specify ports:
+
```

Comment:
nit: wrap to 80col (same below on lines 93 and 96)

Reply:
oops, had the wrapper in 120cols. Fixed :+1: